plat/socionext/uniphier/uniphier_rotpk.S

```diff
@@ -6,6 +6,7 @@
 
 	.global uniphier_rotpk_hash
 	.global uniphier_rotpk_hash_end
+	.section .rodata.uniphier_rotpk_hash, "aS"
```

Comment:
Should this be "a" and not "aS"?

Reply:
You are right.
Looks like I copy-pasted a wrong one.

I googled "S", and it says "section contains zero terminated strings".
This section just contains raw data.

I fixed it and pushed.  Thanks!